src/hooks/useTodos.tsx

```diff
@@ -0,0 +1,62 @@
+import { useCallback, useEffect, useState } from 'react';
+import { StatusFilterOptions } from '../components/StatusFilter';
+import { Todo } from '../types/Todo';
+import { getTodos, TodosError } from '../api/todos';
+
+interface GetFilteredTodosFilter {
+  status: StatusFilterOptions;
+}
+
+function getFilteredTodos(todos: Todo[], filters: GetFilteredTodosFilter) {
+  let filteredTodos = [...todos];
+
+  if (filters.status !== StatusFilterOptions.all) {
+    filteredTodos = filteredTodos.filter(todo => {
+      if (filters.status === StatusFilterOptions.completed) {
+        return todo.completed;
+      } else {
+        return !todo.completed;
+      }
+    });
+  }
+
+  return filteredTodos;
+}
+
+export const useTodos = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [todosLoading, setTodosLoading] = useState(true);
```

Comment:
```suggestion
  const [isLoading, setIsLoading] = useState(fasle);
```
boolean should start with is has are etc...